packages/@okta/vuepress-theme-prose/package.json

```diff
@@ -25,17 +25,17 @@
   "dependencies": {
     "@okta/odyssey": "^0.0.1",
     "@okta/okta-signin-widget": "^5.4.0",
+    "axios": "^0.26.1",
     "build-url": "^6.0.1",
-    "axios": "^0.21.2",
     "cookies.js": "^2.1.15",
     "css-to-json": "^2.1.0",
-    "include-media": "^1.4.9",
-    "jshint": "^2.12.0",
+    "include-media": "^1.4.10",
+    "jshint": "^2.13.4",
     "jwt-decode": "^3.1.2",
-    "moment": "^2.29.2",
-    "moment-timezone": "^0.5.32",
+    "moment": "^2.29.3",
+    "moment-timezone": "^0.5.34",
     "portal-vue": "^2.1.7",
-    "query-string": "^6.13.8",
+    "query-string": "^7.1.1",
```

Comment:
https://github.com/sindresorhus/query-string/releases shows breaking change to `.stringify` but we don't use it. Should be fine.